packages/types/test/unit/executionAddress.test.ts

```diff
@@ -0,0 +1,63 @@
+import {describe, it, expect} from "vitest";
+import {toChecksumAddress} from "../../src/utils/executionAddress.js";
+
+describe("toChecksumAddress", () => {
+  it("should fail with invalid addresses", () => {
+    expect(() => toChecksumAddress("1234")).toThrowError("Invalid address: 1234");
+    expect(() => toChecksumAddress("0x1234")).toThrowError("Invalid address: 0x1234");
+  });
+
+  it("should format address as ERC55", () => {
```

Comment:
since we have multiple test cases

```suggestion
  it("should format addresses as ERC55", () => {
```